core/controllers/schemacontroller/schema_controller.go

```diff
@@ -62,10 +62,7 @@ func Schema(rw http.ResponseWriter, r *http.Request) {
 }
 
 func getSchema() []byte {
-	schema, err := machine.GetSchema()
-	if err != nil {
-		panic(err)
```

Comment:
Shouldn't we handle the error? Not sure what this does

Reply:
I changed the `machine.GetSchema()` function so that it no longer throws an error. I don't think that callers of this function should have to worry about it throwing an error, because it is basically just accessing a field from that package, so I've added an error handling mechanism inside the package itself.